index.js

```diff
@@ -1 +1,33 @@
-console.log('welcome to thirty days of javascript');
\ No newline at end of file
+console.log('welcome to thirty days of javascript');
+
+/*string data type*/
+let myString ="javascript";
+console.log(myJavascript);  //javacript
+
+//string concatenation
+//Connecting two or more strings together is called concatenation. 
+let firstName="abela";
+let lastName="achieng";
+
+let fullName="firstName+space+lastName";
+console.log(fullName);
+
```

Comment:
![image](https://github.com/user-attachments/assets/599219c0-6b59-4abc-9a6f-2c04435c392a)
Here you should use string literals `let fullName=`${firstName} ${lastName}`;` to get the user name and password in the console